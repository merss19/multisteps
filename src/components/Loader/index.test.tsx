import {shallow} from 'enzyme'
import {Loader} from './index'
import * as React from 'react'
import toJson from 'enzyme-to-json'

let props = {
	text: 'Подождите',
}

describe('Component Loader', () => {
	const component = shallow(<Loader {...props}/>)
	it('renders component', () => {
		expect(component).toHaveLength(1)
	})
	it('renders correctly', () => {
		const tree = toJson(shallow(<Loader {...props}/>))
		expect(tree).toMatchSnapshot()
	})
	it('render text', () => {
		expect(component.find('h2').text()).toEqual('Подождите')
	})
})
