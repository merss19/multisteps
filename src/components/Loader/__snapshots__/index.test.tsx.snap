// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Loader renders correctly 1`] = `
<div
  className="text-center"
>
  <h2
    className="pt20"
  >
    Подождите
  </h2>
  <div
    className="loader loader--main"
  />
</div>
`;
